compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -83,11 +83,16 @@ static llvm::cl::opt<bool> enableTripleTilingPipeline(
     llvm::cl::desc("enable triple tiling expert for matmul kernels"),
     llvm::cl::init(false));
 
+// Non-static options are used in other places.
 llvm::cl::opt<std::string> clCPUCodegenTransformDialectFileName(
     "iree-codegen-llvmcpu-use-transform-dialect",
     llvm::cl::desc(
         "MLIR file containing a transform dialect specification to apply"),
     llvm::cl::init(""));
+llvm::cl::opt<bool> clCPUEnableTransformDialectJit(
```

Comment:
If we want to keep supporting both the paths, instead of having an extra flag, maybe use the string value being "jitter" to indicate taking the jitter path. Would be really good to make this an implementation details of the `TransformDialectInterpreterPass` ?

Reply:
The TransformDialectInterpreterPass has quirks related to parsing a file, sharing a module in multithreaded mode and other things we needed to ramp up the no-compilation flow that are not relevant here.
In the fullness of time, the TransformDialectInterpreterPass gets removed and replaced by a builtin database.
Let's not create an artificial coupling please.

Reply:
Alright. I see that the assert is short lived in another PR. Ill stamp this one for now. The combination of flags is somethign we've tried to pay attention to (and failed) but maybe this is how it is for now with these two.

Reply:
thanks, they are mutually exclusive that's for sure